git/util.py

```diff
@@ -173,13 +173,17 @@ class RemoteProgress(object):
     DONE_TOKEN = 'done.'
     TOKEN_SEPARATOR = ', '
 
-    __slots__ = ("_cur_line", "_seen_ops")
+    __slots__ = ("_cur_line", "_seen_ops", "_error_lines")
     re_op_absolute = re.compile(r"(remote: )?([\w\s]+):\s+()(\d+)()(.*)")
     re_op_relative = re.compile(r"(remote: )?([\w\s]+):\s+(\d+)% \((\d+)/(\d+)\)(.*)")
 
     def __init__(self):
         self._seen_ops = list()
         self._cur_line = None
+        self._error_lines = []
+
+    def get_stderr(self):
+        return '\n'.join(self._error_lines)
```

Comment:
I recommend providing the highest-value data available, which in this case in an array of lines. There should be no assumption about how the user will evaluate that data.
As in GitPython in the very majority of cases there is no `get_` prefix, this accessor would be called something like `error_lines()`.